common/_static/custom.css

```diff
@@ -1,27 +1,94 @@
+@font-face {
```

Comment:
@DamianEdwards Not sure if this approach is recommended - we don't do this on the http://asp.net site. Is this to enable consistent fonts on OS X?